flink-table/flink-sql-client/src/test/resources/sql/function.q

```diff
@@ -346,3 +346,69 @@ show user functions;
 SHOW JARS;
 Empty set
 !ok
+
+# ==========================================================================
+# test describe function
+# ==========================================================================
+
+ADD JAR '$VAR_UDF_JAR_PATH';
+[INFO] Execute statement succeeded.
+!info
+
+describe function temp_upperudf;
++-------------------+---------------------------------------------$VAR_UDF_JAR_PATH_DASH+
+|         info name |                                 $VAR_UDF_JAR_PATH_SPACE info value |
```

Comment:
What are these variables? `VAR_UDF_JAR_PATH_SPACE`. Seems some of the docs have them, and others don't but makes it hard to read the source file.

Reply:
Yes i totally agree about it making difficult to read, but this the current standard. The variables are defined here 
https://github.com/apache/flink/blob/master/flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientITCase.java#L148-L168 and this how we handle using the generated temporary jar paths in the tests, while still having correctly formatted tables